.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Open a bug report
+title: "[bug] "
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of the bug or unexpected behavior.
+
+**Steps To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System Informatioon:**
+ - OS: [e.g. Ubuntu 24.04 LTS]
+ - Python Version: [e.g. Python 3.11.2]
+ - Django Version: [e.g. Django 4.2.5]
+ - Browser and Browser Version (if applicable): [e.g. Chromium v126.0.6478.126]
```

Comment:
let's remove the lines about browser and django versions because they don't apply here

Reply:
done @nemesifier